src/IJulia.jl

```diff
@@ -47,6 +47,12 @@ import Logging
 # InteractiveUtils is not used inside IJulia, but loaded in src/kernel.jl
 # and this import makes it possible to load InteractiveUtils from the IJulia namespace
 import InteractiveUtils
+import JSON.json
+@static if isdefined(JSON, :Object)
```

Comment:
I have a very mild preference for using `if pkgversion(...) < v"..."`, but this is also ok if we include a comment.